README.md

```diff
@@ -3,13 +3,16 @@
         height="130">
 </p>
 <p align="center">
+    <a href="https://github.com/badges/shields/graphs/contributors" alt="Contributors">
```

Comment:
More general comment, but what do you think about grouping the badges in separate buckets vs all back to back? For example we could have one line (or paragraph) for activity related badges (contributors, commits, sponsors, backers), another line for dev/code metrics (build status, code coverage, service test status, lgtm alerts), and another line for the social badges.

Reply:
<p align="center">
    <a href="https://github.com/badges/shields/graphs/contributors" alt="Contributors">
        <img src="https://img.shields.io/github/contributors/badges/shields.svg" /></a>
    <a href="#backers" alt="Backers on Open Collective">
        <img src="https://opencollective.com/shields/backers/badge.svg" /></a>
    <a href="#sponsors" alt="Sponsors on Open Collective">
        <img src="https://opencollective.com/shields/sponsors/badge.svg" /></a>
    <a href="https://github.com/badges/shields/pulse" alt="Activity">
        <img src="https://img.shields.io/github/commit-activity/4w/badges/shields.svg" /></a>
</p>
<p align="center">
    <a href="https://circleci.com/gh/badges/shields/tree/master">
        <img src="https://img.shields.io/circleci/project/github/badges/shields/master.svg" alt="build status"></a>
    <a href="https://circleci.com/gh/badges/daily-tests">
        <img src="https://img.shields.io/circleci/project/github/badges/daily-tests.svg?label=service%20tests"
            alt="service-test status"></a>
    <a href="https://coveralls.io/github/badges/shields">
        <img src="https://img.shields.io/coveralls/github/badges/shields.svg"
            alt="coverage"></a>
    <a href="https://lgtm.com/projects/g/badges/shields/alerts/">
        <img src="https://img.shields.io/lgtm/alerts/g/badges/shields.svg"
            alt="Total alerts"/></a>
</p>
<p align="center">
    <a href="https://docs.google.com/spreadsheets/d/1cHIUSVaiKrIFw3KIu0yt-EMNlMkIfU5alE7YKZ4PeOE/edit#gid=0">
        <img src="https://img.shields.io/github/search/badges/shields/extends%20LegacyService.svg?label=legacy%20services%20needing%20refactor"
            alt="legacy services needing refactor"></a>
    <a href="https://github.com/search?utf8=%E2%9C%93&q=previewUrl+repo%3Abadges%2Fshields+path%3A%2Fservices&type=Code&ref=advsearch&l=&l=">
        <img src="https://img.shields.io/github/search/badges/shields/previewUrl.svg?label=`previewUrl`s%20needing%20refactor"
            alt="`previewUrl`s needing refactor"></a>
</p>
<p align="center">
    <a href="https://github.com/badges/shields/compare/gh-pages...master">
        <img src="https://img.shields.io/github/commits-since/badges/shields/gh-pages.svg?label=commits%20to%20be%20deployed"
            alt="commits to be deployed"></a>
    <a href="https://discord.gg/HjJCwm5">
        <img src="https://img.shields.io/discord/308323056592486420.svg?logo=discord"
            alt="chat on Discord"></a>
    <a href="https://twitter.com/intent/follow?screen_name=shields_io">
        <img src="https://img.shields.io/twitter/follow/shields_io.svg?style=social&logo=twitter"
            alt="follow on Twitter"></a>
</p>

Reply:
I suppose that does end up taking up a lot more vertical real estate so not crazy about that 🤔 I just get used to locating a badge (like coverage or service tests in particular) in a particular location  and was trying to think of a way they wouldn't jump around so much when we add new badges

Reply:
We are getting a little bit into https://twitter.com/NikolayTalanov/status/860444084594393088 territory 😁 

We could group them into different sections in the readme, like put stuff related to community participation in one place, and stuff related to quality in another. It could be worth having a "maintainers dashboard" in one of the other readme files.

Reply:
Ah that gave me a good chuckle. I like the idea of a maintainers view too. The ones I check the most are the coverage and build/service test status so would be nice to have somewhere to find those quickly.